rules/CodeQuality/Rector/FunctionLike/RemoveAlwaysTrueConditionSetInConstructorRector.php

```diff
@@ -90,81 +87,50 @@ public function go()
      */
     public function getNodeTypes(): array
     {
-        return [ClassMethod::class, Closure::class];
+        return [If_::class];
     }
 
     /**
-     * @param ClassMethod|Closure $node
+     * @param If_ $node
+     * @return Stmt[]|null
      */
-    public function refactor(Node $node): ?Node
+    public function refactor(Node $node): ?array
     {
-        if ($node->stmts === null) {
+        $ifStmt = $this->matchTruableIf($node);
+        if (! $ifStmt instanceof If_) {
             return null;
         }
 
-        if ($node->stmts === []) {
+        if ($ifStmt->stmts === null) {
```

Comment:
It needs compare with `[]`, if stmts never be null

Reply:
👍